src/ontology/foodon-edit.owl

```diff
@@ -98634,6 +98823,9 @@ AnnotationAssertion(dc:contributor obo:FOODON_03304292 "https://orcid.org/0000-0
 AnnotationAssertion(dc:contributor obo:FOODON_03304292 "https://orcid.org/0000-0002-8844-9165")
 AnnotationAssertion(dc:contributor obo:FOODON_03304292 "https://orcid.org/0000-0002-9578-0788")
 AnnotationAssertion(rdfs:comment obo:FOODON_03304292 "Spice mixes can be whole or in parts, but are most commonly blended spices or herbs."@en)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03305145 "A farm-produced crop, usually fruit or vegetable, that is fresh and in the same state as where and when it was harvested."@en)
```

Comment:
I think term ids collided here.  You are defining produce, but there seems to be an existing term about spice mixes at same id?

Reply:
I haven't created a new term for any of them. Just updated the existing term 'produce (raw)' which has ID 'FOODON_03305145', while 'spice mixture (whole or parts)' has FOODON:03304292.

Reply:
Woops, my bad!  I thought the annotations were about the same class, but I see I was wrong about that.  

But I see that this is an edit on foodon-edit.owl .  The edit could go into foodon-product-import.owl